src/Psalm/Internal/Analyzer/FunctionLikeAnalyzer.php

```diff
@@ -719,6 +725,8 @@ public function analyze(
             }
 
             if (!$is_expected) {
+                assert(class_exists($possibly_thrown_exception));
```

Comment:
should i keep this assert or remove it, it makes sense for throws keyword to only contain classes which exist

Reply:
ideally this check would not be needed, but that means modifying the return type for the function `getUncaughtThrows` which would be a bigger change

Reply:
Please drop it. It's risky because classes could be stubbed or inserted through a Plugin so you can't guarantee they'll exists for Psalm to autoload